client/assets/js/main.js

```diff
@@ -21,4 +21,54 @@ const shortenURL = () => {
       document.getElementById('shortened-url-lists').innerHTML = data.message
   })
 }
+
 document.querySelector('.create-shorten-url').onclick = shortenURL
+
+renderurls(urls)
+
+//===================================
+const verifyURLs = (e) => {
+	if(e.target.elements.shortenedURL.value !== '' && e.target.elements.shortenedURL.value.length < 7){
+		console.log('Too short.')
+		return 1 //1 means the short URL is too short.
+	} else if(e.target.elements.originalURL.value.length >= 0) {
+		return fetch(e.target.elements.originalURL.value, { mode: 'no-cors' })
```

Comment:
Using the above method should also solve some of the issues I think you're encountering regarding fetch. As you're seeing, fetch doesn't really return a normal value - it returns a Promise for a value. There are some ways to handle this (like using the async/await pattern to make your code more synchronous) but you should be able to get away with writing a state holding object (i.e., the `verificationStatus`)